doc/data/messages/l/lost-exception/bad.py

```diff
@@ -0,0 +1,7 @@
+def fruit(value):
+    try:
+        return 2 / value
+    except ZeroDivisionError:
+        print('Connot divide zero.')
```

Comment:
Here the issue is that the exception is not re-raised, so it's not swallowed and does not illustrate the issue.